yarn.lock

```diff
@@ -20601,9 +20601,9 @@ ssri@^8.0.1:
   dependencies:
     minipass "^3.1.1"
 
-"ssv-keys@https://github.com/bloxapp/ssv-keys.git":
-  version "1.0.4"
-  resolved "https://github.com/bloxapp/ssv-keys.git#dbd59394f702fb00d7836f7d19dd92925b4e353f"
+"ssv-keys@https://github.com/bloxapp/ssv-keys.git#1.0.7":
```

Comment:
1.0.9

Reply:
it's in progress
will deliver it in new PR